website/config.yaml

```diff
@@ -65,8 +65,9 @@ params:
         url: 'https://slack.k8s.io/'
         icon: fab fa-slack
         desc: 'Chat with us on Slack in the #aws-provider channel'
-  latest_release_version: v0.6.2
+  latest_release_version: v0.6.3
   versions:
+    - "v0.6.3"
     - "v0.6.2"
```

Comment:
Should we remove v0.6.2 and v0.6.1 since they're known bad?

Reply:
agree.